SimplenoteIntents/IntentHandler.swift

```diff
@@ -51,13 +43,45 @@ extension IntentHandler: NoteWidgetIntentHandling {
     }
 
     func defaultNote(for intent: NoteWidgetIntent) -> WidgetNote? {
-        guard let dataController = try? WidgetDataController(coreDataManager: coreDataManager) else {
+        guard let note = try? WidgetDataController(context: coreDataManager.managedObjectContext).firstNote() else {
```

Comment:
Another Q: What's the `IntentHandler` lifecycle like? (does it get instantiated when the Widget goes live, then it gets killed?)

I've notice nearly all of the API(s) in here require a `WidgetDataController` instance. Maybe we should instantiate that in the `init`, and keep it around?